test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -399,6 +399,19 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result = ExecuteWebCommand -command $command
         $result.Error | Should BeNullOrEmpty
     }
+
+    It "Validate Invoke-WebRequest returns HTTP errors in exception" {
+
+        $command = "Invoke-WebRequest -Uri http://httpbin.org/status/418"
+        $result = ExecuteWebCommand -command $command
+
+        $result.Error.ErrorDetails.Message | Should Match "\-=\[ teapot \]"
+        $result.Error.Exception | Should BeOfType Microsoft.PowerShell.Commands.HttpResponseException
+        $result.Error.Exception.Response.StatusCode | Should Be 418
+        $result.Error.Exception.Response.ReasonPhrase | Should Be "I'm a teapot"
+        $result.Error.Exception.Message | Should Be "Response status code does not indicate success: 418 (I'm a teapot)."
```

Comment:
Yes, I think so, `$result.Error.Exception.Message | Should BeLike * 418 (I'm a teapot)` should be good, since they are returned from the Http service, not affected by PS localization.